Stripe/STPPaymentIntentAction.m

```diff
@@ -0,0 +1,63 @@
+//
+//  STPPaymentIntentAction.m
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/8/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPaymentIntentAction.h"
+
+#import "STPPaymentIntent+Private.h"
+#import "STPPaymentIntentActionRedirectToURL.h"
+
+#import "NSDictionary+Stripe.h"
+
+@interface STPPaymentIntentAction()
+
+@property (nonatomic) STPPaymentIntentActionType type;
+@property (nonatomic, strong, nullable) STPPaymentIntentActionRedirectToURL* redirectToURL;
+@property (nonatomic, copy, nonnull, readwrite) NSDictionary *allResponseFields;
+
+@end
+
+@implementation STPPaymentIntentAction
```

Comment:
Looks like this is missing the `description` override from `STPPaymentIntentSourceAction`